lib/hocon/impl/simple_config.rb

```diff
@@ -61,12 +61,12 @@ def self.find_key(me, key, expected, original_path)
     if v.value_type == ConfigValueType::NULL
       raise ConfigNullError.new(v.origin,
                                 (ConfigNullError.make_message(original_path.render,
-                                                              (not expected.nil?) ? ConfigValueType.value_type_name(expected) : nil)),
+                                                              (not expected.nil?) ? Hocon::ConfigValueType.value_type_name(expected) : nil)),
                                 nil)
     elsif (not expected.nil?) && v.value_type != expected
       raise ConfigWrongTypeError.new(v.origin,
-                                     "#{original_path.render} has type #{ConfigValueType.value_type_name(v.value_type)} " +
-                                         "rather than #{ConfigValueType.value_type_name(expected)}",
+                                     "#{original_path.render} has type #{Hocon::ConfigValueType.value_type_name(v.value_type)} " +
+                                         "rather than #{Hocon::ConfigValueType.value_type_name(expected)}",
```

Comment:
Oh yeah, there's quite a few. I'm going through and changing them all right now.